api-admin/src/main/java/bio/terra/pearl/api/admin/models/dto/ProfileUpdateDto.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.pearl.api.admin.models.dto;
+
+import bio.terra.pearl.core.model.participant.Profile;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ProfileUpdateDto {
+  private String justification;
```

Comment:
That word choice comes directly from Parker, which is why Erin and I were using it. Erin said CMI uses the term, and I think it also helps disambiguate from a "Participant Note". But I get what you're saying. Open to changing but since it wasn't my wording I feel a bit hesitant